src/Faithlife.Testing/AssertEx.cs

```diff
@@ -842,7 +836,47 @@ private static string ToString(object obj)
 			return toString;
 		}
 
-		private static readonly JsonSettings s_jsonWithTabs = new JsonSettings { IsIndented = true };
+		private static string ToPrettyJson(JToken token, int tabLevel)
+		{
+			var indent = new string('\t', tabLevel);
+			if (token.Type == JTokenType.Array)
```

Comment:
Formatting nit: Instead of these chained `if`/`return` blocks, this would communicate the structure of the method more cleanly as a `switch` block on `token.Type`.